### PR TITLE
Add support for multiple result files

### PR DESCRIPTION
**Pull request by ErikSchierboom**
This PR adds support for the tooling webserver to return multiple result files, which is required for the representer as it outputs both `representation.txt` and `mapping.json` files.

There is a dependency on changes to the `tooling-invoker` and `tooling-orchestrator`, for which I'll submit PR's later.

See https://github.com/exercism/development-environment/issues/34

### CODE REVIEW

File: src/tooling_webserver.nim [hunk 1]
```diff
@@ -40,12 +41,12 @@ router routes:
...
         {
             "exit_status": exit_status,
-            "result": nil
+            "results": %*{}
         }
 
-    let results_filepath = request.params["results_filepath"]
-    if fileExists(fmt"{output_dir}/{results_filepath}"):
-        response["result"] = % readFile(fmt"{output_dir}/{results_filepath}")
+    for results_filepath in request.params["results_filepaths"].split(','):
```
Comment (ErikSchierboom) [inline src/tooling_webserver.nim]: As far as I could tell by checking the Jester docs, the `request.params` property is a mapping from strings to strings. This means that I couldn't pass in the result file paths as an array, so I concatenated them using the `,` character.

File: src/tooling_webserver.nim [hunk 2]
```diff
@@ -40,12 +41,12 @@ router routes:
     let response = %*
         {
             "exit_status": exit_status,
-            "result": nil
+            "results": %*{}
```
Comment (ErikSchierboom) [inline src/tooling_webserver.nim]: We made this a JObject to allow key/value pairs to be added later on for the results file paths to results file paths contents mapping.

File: src/tooling_webserver.nim [hunk 3]
```diff
@@ -2,6 +2,7 @@ import jester
 import os
 import osproc
 import strformat
+import strutils
```
Comment (ErikSchierboom) [inline src/tooling_webserver.nim:5]: This is required for the `split()` procedure that is used later

File: src/tooling_webserver.nim [hunk 4]
```diff
@@ -40,12 +41,12 @@ router routes:
     let response = %*
         {
             "exit_status": exit_status,
-            "result": nil
+            "output_files": %*{}
```
Comment (ErikSchierboom) [inline src/tooling_webserver.nim:44]: I've renamed this to `output_files` which IMHO better indicate what its contents are. It also fits with the input/output dir terminology used to describe the tooling interfaces.

Comment (iHiD) [reply src/tooling_webserver.nim:44]: Cool. I've commented on this [here](https://github.com/exercism/tooling-orchestrator/pull/14#pullrequestreview-492214423) but I'm happy with it 🙂

**Review by iHiD — APPROVED** [review]

**Pull request merged**